draft-sharabayko-mops-srt.md

```diff
@@ -596,57 +679,100 @@ Last Acknowledged Packet Sequence Number (32 bits):
-data packet deliveries. The receiver notifies the sender about lost data packets sending the NAK packets.
-The NAK packet contains a list of sequence numbers of lost packets.
+Negative acknowledgement (NAK) control packets are used to signal failed data packet 
+deliveries. The receiver notifies the sender about lost data packets by sending a NAK 
+packet that contains a list of sequence numbers for those lost packets.
+
+An SRT NAK packet is formatted as follows:
+
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ---
+|1|        Control Type         |           Reserved            |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                   Type-specific Information                   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ SRT Header
+|                           Time Stamp                          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                     Destination Socket ID                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ---
+|0|                 Lost packet sequence number                 |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|1|                    List of lost packets                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ CIF (Loss List)
+|0|                            Up to                            |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|0|                 Lost packet sequence number                 |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ---
 
 Control Type: 
-: The type value of NAK control packet is "3".
+: The type value of a NAK control packet is "3".
 
 Type-specific Information: 
 : This field is reserved for future definition.
 
-Control Information Field:
-: A single value or a list of lost packets sequence numbers. See packet sequence number coding in
-  {{packet-seq-list-coding}}.
+Control Information Field (CIF):
+: A single value or a list of lost packets sequence numbers. See packet sequence number 
+coding in {{packet-seq-list-coding}}.
+
 
 ### Shutdown {#ctrl-pkt-shutdown}
 
 Shutdown control packets are used to initiate the closing of an SRT connection.
 
+An SRT SHUTDOWN Control packet is formatted as follows:
+
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ---
+|1|        Control Type         |           Reserved            |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                   Type-specific Information                   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ SRT Header
```

Comment:
This line is 77 characters in width. The conversion to IETF txt format requires 72.
Maybe we can fix it this way:
```
+-+-+-+-+-+-+-+-+SRT Header-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
```